### PR TITLE
Jetpack connect: Use named exports

### PR DESCRIPTION
**Pull request by sirreal**
This PR fixes the way functions are exported from `jetpack-connect/actions`. See p4TIVU-8Lf-p2

Only named imports are used, so this PR correct the default export with named exports.

## Testing
1. Tests pass
1. Try connection flows
1. Verify that only named imports are used:
   ```sh
   ack -B10 'jetpack-connect/actions' client
   ```
   All the imports should should be inside braces.

### CODE REVIEW

File: client/state/jetpack-connect/actions.js [hunk 1]
```diff
@@ -63,493 +63,504 @@ const remoteAuthPath =
...
-		};
-	},
+export function confirmJetpackInstallStatus( status ) {
+	return dispatch => {
+		dispatch( {
+			type: JETPACK_CONNECT_CONFIRM_JETPACK_STATUS,
+			status: status,
+		} );
+	};
+}
```
Comment (ockham) [inline client/state/jetpack-connect/actions.js]: Hmm, there's absolutely no reason for this to be a thunk -- this can be rewritten as a perfectly fine synchronous action. 😕 

```js
export function confirmJetpackInstallStatus( status ) {
	return( {
		type: JETPACK_CONNECT_CONFIRM_JETPACK_STATUS,
		status,
	} );
}
```

Since it's not related to the goal of this PR, and there a test for it, I guess we should probably do it in a separate PR. Still hurts my eyes...

File: client/state/jetpack-connect/actions.js [hunk 2]
```diff
@@ -63,493 +63,504 @@ const remoteAuthPath =
...
-	dismissUrl( url ) {
-		return dispatch => {
+export function dismissUrl( url ) {
+	return dispatch => {
+		dispatch( {
+			type: JETPACK_CONNECT_DISMISS_URL_STATUS,
+			url: url,
+		} );
+	};
+}
```
Comment (ockham) [inline client/state/jetpack-connect/actions.js]: Same as https://github.com/Automattic/wp-calypso/pull/20294/files#r153581388

File: client/state/jetpack-connect/actions.js (hunk 1 above)
Comment (sirreal) [reply client/state/jetpack-connect/actions.js]: Yep, there are many in this file. I'll follow up to clean that up 👍

Comment (sirreal) [reply client/state/jetpack-connect/actions.js]: see #20324
